docs/user/contacts.md

```diff
@@ -0,0 +1,15 @@
+# Contact Us
+
+Welcome to the Vald project!!
+
+Vald is an open source project and we would like to hear from you.
+
+If you are interested in Vald, or you have an any question about Vald, please feel free to contact us.
```

Comment:
```suggestion
If you are interested in Vald, or you have any questions about Vald, please feel free to contact us.
```

Reply:
Thank you for your review.
I will fix it soon